plugins/inputs/intel_powerstat/README.md

```diff
@@ -0,0 +1,199 @@
+# Intel PowerStat Input Plugin
+
+Telemetry frameworks allow users to monitor critical platform level metrics. 
+Key source of platform telemetry is power domain that is beneficial for MANO/Monitoring&Analytics systems 
+to take preventive/corrective actions based on platform busyness, CPU temperature, actual CPU utilization 
+and power statistics. Main use cases are power saving and workload migration.
+
+Intel PowerStat plugin supports Intel based platforms and assumes presence of Linux based OS.
+
+### Configuration:
+```toml
+# Intel PowerStat plugin enables monitoring of platform metrics (power, TDP) and per-CPU metrics like temperature, power and utilization.
+[[inputs.intel_powerstat]]
+  ## All global metrics are always collected by Intel PowerStat plugin.
+  ## User can choose which per-CPU metrics are monitored by the plugin in cpu_metrics array.
+  ## Empty array means no per-CPU specific metrics will be collected by the plugin - in this case only platform level
+  ## telemetry will be exposed by Intel PowerStat plugin.
+  ## Supported options:
+  ## "cpu_frequency", "cpu_c1_state_residency", "cpu_c6_state_residency", "cpu_busy_cycles", "cpu_temperature", "cpu_busy_frequency"
+  # cpu_metrics = []
+```
+### Example: Configuration with no per-CPU telemetry
+This configuration allows getting global metrics (processor package specific), no per-CPU metrics are collected:
+```toml
+[[inputs.intel_powerstat]]
+  cpu_metrics = []
```

Comment:
This can also be removed, can't it?

Reply:
Yes, that's right. But it shows explicitly what is the result of putting empty array in config. Is that bad?

Reply:
No it's ok, but it suggests you need to specify it to be empty. Fine if you want to keep it if you want to make it explicit.

Reply:
It will be exactly the same behavior if we do not specify this empty array - no per-CPU metrics gathered. 
As guided here: https://github.com/influxdata/telegraf/wiki/Review#configuration: `have one # for defaults, which should always match the default value of the plugin` :)

Reply:
Example without array also added to `README.md`